hudi-gcp/src/main/java/org/apache/hudi/gcp/bigquery/BigQuerySyncConfig.java

```diff
@@ -122,6 +121,16 @@ public class BigQuerySyncConfig extends HoodieSyncConfig implements Serializable
       .markAdvanced()
       .withDocumentation("Fetch file listing from Hudi's metadata");
 
+  public static final ConfigProperty<Boolean> BIGQUERY_SYNC_REQUIRE_PARTITION_FILTER = ConfigProperty
+      .key("hoodie.gcp.bigquery.sync.require_partition_filter")
+      .defaultValue(false)
+      .withDocumentation("If true, configure table to require a partition filter to be specified when querying the table");
+
+  public static final ConfigProperty<String> BIGQUERY_SYNC_BIG_LAKE_CONNECTION_ID = ConfigProperty
+      .key("hoodie.onehouse.gcp.bigquery.sync.big_lake_connection_id")
+      .noDefaultValue()
+      .withDocumentation("The Big Lake connection ID to use");
+
```

Comment:
Add `.sinceVersion("0.14.0")` and mark them advanced (`.markAdvanced()`, if they are not required to be set by the users)?

Reply:
should it be 0.14.1?

Reply:
yeah